packages/itwinui-react/src/core/Menu/MenuItemSkeleton.tsx

```diff
@@ -50,9 +51,8 @@ export const MenuItemSkeleton = (props: MenuItemSkeletonProps) => {
   useGlobals();
 
   return (
-    <li
+    <ListItem
```

Comment:
Is it possible to avoid using `ListItem` here? `ListItem` is the new api, whereas `MenuItemSkeleton` was done in a very weird way with overrides and has undesirable generic class names like `iui-content`.

Reply:
Using Box

Reply:
the `iui-list-` classes are still used though. What I meant was keep the old classes (basically left side of [this diff](https://github.com/iTwin/iTwinUI/pull/1295/commits/7b1ec0d9b4d5406480877579e0cb84a30e57c48d)) and update the css selectors to target those old classes.
```diff
- .iui-list-item-content {
+ :is(.iui-list-item-content, .iui-menu-item-skeleton > .iui-content) {
```

and then in a future PR, we can improve the css for it.

Reply:
Changed back classnames and css styles

Reply:
oops, fixed